stone/backends/python_client.py

```diff
@@ -380,7 +392,14 @@ def _generate_docstring_for_func(self, namespace, arg_data_type,
         :param str footer: Additional notes at the end of the docstring.
         """
         fields = [] if is_void_type(arg_data_type) else arg_data_type.fields
-        if not fields and not overview:
+
+        attrs_list = []
```

Comment:
nit: `attr_values` or something might be more descriptive.

Reply:
Updated to `attr_lines` since they are each a line of text.